libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java

```diff
@@ -26,26 +26,37 @@
  */
 package com.salesforce.androidsdk.phonegap;
 
-import java.util.Arrays;
-import java.util.List;
+import android.support.test.filters.MediumTest;
+import android.support.test.runner.AndroidJUnit4;
 
-import com.salesforce.androidsdk.phonegap.util.test.JSTestCase;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
 
+import java.util.Arrays;
+import java.util.List;
 
 /**
- * Running javascript tests for SmartStore plugin
+ * Running javascript tests for SmartStore plugin.
  */
+@RunWith(AndroidJUnit4.class)
+@MediumTest
```

Comment:
Shouldn’t it be large?

Reply:
Not sure if it's based on runtime of the suite or number of tests in the suite. I can change it to large.

Reply:
Number of tests is something the runner can easily figure out. It has to do with the runtime characteristics of the tests. Here is an old blog post about it: https://testing.googleblog.com/2010/12/test-sizes.html?m=1